docs/03-apps-and-frameworks.md

```diff
@@ -0,0 +1,18 @@
+
+### Laravel
+  * [Account](http://account.stjude.cloud) / [Github](https://github.com/stjudecloud/account)
+  * [Viz Community](https://viz.stjude.cloud) / [Github](https://github.com/stjude/viz)
+  * [PeCan](https://pecan.stjude.cloud) / [Github](https://github.com/stjude/pecan) / `Uses Vue for Front-End`
```

Comment:
I think we should be consistent in how call out Vue. For Viz, it's listed in both under Laravel and Vue but PeCan has a note. Additionally, pending how we treat the stacks, I'd note that Platform uses Rails on the backend.

Reply:
@sawiggins I agree, I've added Viz and PeCan to both of their front and backend frameworks. I think this may be better in the next revision to list each app, and then which frameworks they use, instead of listing by framework. Next time?

I've also added links to each frameworks homepage for convenience and to be consistent with using links for each app/resource.